README.md

```diff
@@ -1,8 +1,87 @@
-# `@ubiquity/ts-template`
+# `@ubiquity/rpc-handler`
 
-This template repository includes support for the following:
+This packages leverages [Chainlist's](https://github.com/DefiLlama/chainlist) network RPC list to return the lowest latency provider from the list for any given network ID.
 
-- TypeScript
-- Environment Variables
-- Conventional Commits
-- Automatic deployment to Cloudflare Pages
+## Features
+
+- Returns the lowest latency provider for a given network ID
+- Drops bad endpoints from the list and creates a runtime/local storage cache
+- Can re-test the cached RPCs by calling `handler.getFastestRpcProvider()`
+- Can be used in both the browser and Node.js
+- Fully configurable and extendable
+- Only uses endpoints which Chainlist report as tracking _no_ data (see [`extraRpcs.js`](https://github.com/DefiLlama/chainlist/blob/main/constants/extraRpcs.js))
+
+## Installation
+
+```bash
+yarn add @ubiquity/rpc-handler
```

Comment:
remame here too accordingly

Reply:
if people do yarn add @ubiquity/ will get 404, do `"@ubiquitydao/rpc-handler"` considering the package it's gonna be hosted there, i rember i renamed just to match the repo name